src/store/connection/actions.js

```diff
@@ -76,3 +100,38 @@ export const getConnectionsCount = () => async (dispatch, getState) => {
     dispatch(fetchFailure());
   }
 };
+
+export const newConnectionAction = () => async (dispatch, getState) => {
+  const {
+    connections: {
+      host,
+      port,
+      nameDB,
+      username: name,
+      password,
+      type,
+      nameConnection,
+    }
+  } = getState();
+
+  if (!name || !password || !host || !port || !nameDB || !type || !nameConnection) {
```

Comment:
Not the best way to check fields
It would be better to  have an array of fields and use some find or etc method